src/components/ModalPayment/ModalAmount/ModalAmount.tsx

```diff
@@ -130,7 +168,61 @@ export const Modal = (props: Props) => {
           </ErrorMessage>
         )}
       </AmountContainer>
-
+      {amount >= 45 && (
+        <AmountContainer>
+          <Subheader>{t('paymentProcessing.amount.adopt_lantern')}</Subheader>
+          <label>{t('paymentProcessing.amount.tier_2_donation')}</label>
+          <br />
+          <br />
+          <label>{t('paymentProcessing.amount.personalize')}</label>
+          <br />
+          <br />
+          <SingleRowFormat>
+            <RowFormat width="38%">
+              <LabelText htmlFor="name">
+                {t('paymentProcessing.amount.labels.first_name')}
+              </LabelText>
+              <InputText
+                name="first_name"
+                type="text"
+                className="modalInput--input"
+                onChange={handleOnChange}
+                placeholder={t(
+                  'paymentProcessing.amount.place_holder.first_name'
+                )}
+              />
+            </RowFormat>
+            <RowFormat width="20%" mobileWidth="30%">
+              <LabelText htmlFor="email">
```

Comment:
Why `htmlFor="email"` and below but not the first name?

Reply:
these were just copy and pasted from earlier, will cleanup